translationRecorder/app/src/main/assets/chunks/ot/ecc/chunks.json

```diff
@@ -1 +1 @@
s": "6"}, {"firstvs": "7", "id": "06-07", "lastvs": "8"}, {"firstvs": "9", "id": "06-09", "lastvs": "12"}, {"firstvs": "1", "id": "07-01", "lastvs": "2"}, {"firstvs": "3", "id": "07-03", "lastvs": "4"}, {"firstvs": "5", "id": "07-05", "lastvs": "6"}, {"firstvs": "7", "id": "07-07", "lastvs": "7"}, {"firstvs": "8", "id": "07-08", "lastvs": "9"}, {"firstvs": "10", "id": "07-10", "lastvs": "10"}, {"firstvs": "11", "id": "07-11", "lastvs": "12"}, {"firstvs": "13", "id": "07-13", "lastvs": "13"}, {"firstvs": "14", "id": "07-14", "lastvs": "14"}, {"firstvs": "15", "id": "07-15", "lastvs": "16"}, {"firstvs": "17", "id": "07-17", "lastvs": "18"}, {"firstvs": "19", "id": "07-19", "lastvs": "20"}, {"firstvs": "21", "id": "07-21", "lastvs": "22"}, {"firstvs": "23", "id": "07-23", "lastvs": "25"}, {"firstvs": "26", "id": "07-26", "lastvs": "26"}, {"firstvs": "27", "id": "07-27", "lastvs": "28"}, {"firstvs": "29", "id": "07-29", "lastvs": "29"}, {"firstvs": "1", "id": "08-01", "lastvs": "1"}, {"firstvs": "2", "id": "08-02", "lastvs": "4"}, {"firstvs": "5", "id": "08-05", "lastvs": "7"}, {"firstvs": "8", "id": "08-08", "lastvs": "9"}, {"firstvs": "10", "id": "08-10", "lastvs": "11"}, {"firstvs": "12", "id": "08-12", "lastvs": "13"}, {"firstvs": "14", "id": "08-14", "lastvs": "15"}, {"firstvs": "16", "id": "08-16", "lastvs": "17"}, {"firstvs": "1", "id": "09-01", "lastvs": "1"}, {"firstvs": "2", "id": "09-02", "lastvs": "2"}, {"firstvs": "3", "id": "09-03", "lastvs": "3"}, {"firstvs": "4", "id": "09-04", "lastvs": "5"}, {"firstvs": "6", "id": "09-06", "lastvs": "8"}, {"firstvs": "9", "id": "09-09", "lastvs": "10"}, {"firstvs": "11", "id": "09-11", "lastvs": "12"}, {"firstvs": "13", "id": "09-13", "lastvs": "15"}, {"firstvs": "16", "id": "09-16", "lastvs": "16"}, {"firstvs": "17", "id": "09-17", "lastvs": "18"}, {"firstvs": "1", "id": "10-01", "lastvs": "3"}, {"firstvs": "4", "id": "10-04", "lastvs": "4"}, {"firstvs": "5", "id": "10-05", "lastvs": "7"}, {"firstvs": "8", "id": "10-08", "lastvs": "9"}, {"firstvs": "10", "id": "10-10", "lastvs": "11"}, {"firstvs": "12", "id": "10-12", "lastvs": "12"}, {"firstvs": "13", "id": "10-13", "lastvs": "14"}, {"firstvs": "15", "id": "10-15", "lastvs": "15"}, {"firstvs": "16", "id": "10-16", "lastvs": "17"}, {"firstvs": "18", "id": "10-18", "lastvs": "19"}, {"firstvs": "20", "id": "10-20", "lastvs": "20"}, {"firstvs": "1", "id": "11-01", "lastvs": "3"}, {"firstvs": "4", "id": "11-04", "lastvs": "5"}, {"firstvs": "6", "id": "11-06", "lastvs": "8"}, {"firstvs": "9", "id": "11-09", "lastvs": "10"}, {"firstvs": "1", "id": "12-01", "lastvs": "2"}, {"firstvs": "3", "id": "12-03", "lastvs": "3"}, {"firstvs": "4", "id": "12-04", "lastvs": "4"}, {"firstvs": "5", "id": "12-05", "lastvs": "5"}, {"firstvs": "6", "id": "12-06", "lastvs": "7"}, {"firstvs": "8", "id": "12-08", "lastvs": "9"}, {"firstvs": "10", "id": "12-10", "lastvs": "11"}, {"firstvs": "12", "id": "12-12", "lastvs": "12"}, {"firstvs": "13", "id": "12-00", "lastvs": "14"}]
```

Comment:
why is this changed while the rest are removed?

Reply:
@sarabiaj these are updated from unfoldingword api, by running python scripts included into the project. The rest are deleted, because they were just duplicates